doc/continuous_deployment/Run_gerrit_pipeline_on_github_PR.md

```diff
@@ -0,0 +1,24 @@
+# Run gerrit pipeline on github PR
+
+When to do it:
+
+You introduce a change to github/gerrit handling code, e.g. the
+trigger mechanism - code that depends on Zuul or other Jenkins trigger plugins
+
+## 
+
+1. Clone Jenkins job `winci-server2016-devel` to some temporary `tmp-devel-gerrit-check`
+2. Configure `tmp-devel-gerrit-check`:
+* Properties content: BRANCH_NAME={branch to test}
+* Branches to build: {branch to test}
+3. Schedule `tmp-devel-gerrit-check` job manually with parameters:
+* `ZUUL_PROJECT=Juniper/contrail-controller`
+* `ZUUL_BRANCH=master`
+* `ZUUL_REF=None`
+* `ZUUL_URL=http://10.84.12.75/merge-warrior`
+* `ZUUL_UUID=$someRandomUUID`
```

Comment:
And yet you used `NO_ZUUL_UUID_SET` in your job. So which one is it - random uuid or NO_ZUUL_UUID_SET?

Reply:
I don't think it matters but you should decide on one - not write something in docs and use it some other way. If there's no need for developer to generate UUID then you should say that in docs.

Reply:
It does. Log path on log server is constructed from `ZUUL_UUID`.

Reply:
Here are your logs: http://logs.opencontrail.org/winci/NO_ZUUL_UUID_SET/

Reply:
Sorry for spam as it's not related to this PR, but we should change this behaviour. I mean, maybe random generated log path would be better. Then `ZUUL_UUID` would have more clearly defined purpose.

Reply:
What should I put there then? @sodar

Reply:
I would go with generating random uuid.